src/server-middleware/healthcheck.js

```diff
@@ -0,0 +1,4 @@
+export default function healthcheck(req, res) {
+  res.setHeader('Content-Type', 'text/plain')
+  res.end('OK')
```

Comment:
It's recommended to use `.end` when [not returning any data](https://expressjs.com/en/api.html#res.end). To send data, it's better to use `.send` instead. Returning an object like the following would make the response identical to API and ingestion-server.

Also `.send` sets the `Content-Type` header automatically!

```suggestion
  res.send({ status: '200 OK' })
```

Reply:
After some research: The `res` here is the Node.js http response object: https://github.com/senchalabs/connect#appuseroute-fn (not express), so using `res.send` causes a `res.send is not a function` error.

Reply:
Ooooh, that's interesting! I was under the impression that Nuxt was running Express under the hood. While we can still do something like
```suggestion
  res.setHeader('Content-Type', 'application/json')
  res.end(JSON.stringify({ status: '200 OK' }))
```
it might be too much effort for a response that doesn't even matter to a health-check service.

Reply:
I hope @rbadillap can clarify if plaintext 'OK' response is enough, or JSON is better?

Reply:
The sole purpose of this report is to reduce the size of the response, so from a DevOps perspective there is no difference between the two solutions as we are achieving the main goal.
Thank you for tackling this so quick.